tests/MailboxTest.php

```diff
@@ -102,6 +102,8 @@ public function testGetMessageSequenceThrowsException(): void
     {
         $this->expectException(InvalidSearchCriteriaException::class);
         $this->mailbox->getMessageSequence('-1:x');
+
+        $this->expectExceptionMessageRegExp('/Syntax error in sequence/');
```

Comment:
This line is never reached: you have to put it before the `getMessageSequence` call

Reply:
You are right, fixed 👍

Reply:
I don't see any changes, the tested code is still above the expectation

Reply:
My fault, haven't pushed the commit 🤦‍♂